src/Logic.hs

```diff
@@ -570,3 +609,15 @@ handleEvent
   -> Action ProjectState
 handleEvent triggerConfig event state =
   handleEventInternal triggerConfig event state >>= proceedUntilFixedPoint
+
+
+-- TODO this is very Channable specific, perhaps it should be more generic
+parseVersion :: TagName -> Maybe Integer
+parseVersion (TagName t) = (\case ('v', num) -> parseNum num; _ -> Nothing) =<< Text.uncons t
```

Comment:
Nit: 

```haskell
parseVersion (TagName t) = Text.uncons t >>= \case
  ('v', num) -> parseNum num
  _ -> Nothing
```

is easier to read (simply from left to right, top to bottom, rather than alternating left and right from both ends)

Reply:
Ah, yes, much better

Reply:
Fixed